server/server/urls.py

```diff
@@ -36,18 +36,41 @@
          name='password_reset_complete'),
 ]
 
+
 # zentral apps
-for app_name, app_config in zentral_settings.get('apps', {}).items():
-    app_shortname = app_name.rsplit('.', 1)[-1]
-    for url_prefix, url_module_name in (("", "urls"),
-                                        ("api/", "api_urls"),
-                                        ("metrics/", "metrics_urls")):
-        if url_module_name == "metrics_urls" and not app_config.get("metrics", False):
-            continue
-        try:
-            urlpatterns.append(path(f"{url_prefix}{app_shortname}/", include(f"{app_name}.{url_module_name}")))
-        except ModuleNotFoundError:
-            pass
+def build_urlpatterns_for_zentral_apps():
```

Comment:
This change is already merged on main

Reply:
Git will be smart enough to know it. I can rebase with `main` if you fancy.

Reply:
yes please

Reply:
Done.